docs/styles/_base.scss

```diff
@@ -25,26 +27,12 @@ body {
     }
 }
 
-a, a:visited {
-    color: var(--primary);
-    padding: 2px;
-    transition-property: background-position-x, color, opacity;
-    transition-duration: 0.2s;
-    transition-timing-function: linear;
-    background: linear-gradient(
-        80deg,
-        var(--bg, var(--black)) 33%,
-        var(--primary),
-        var(--secondary)
-    );
-    background-size: 300% 100%;
-    background-position-x: 0;
+main {
+    margin: 1rem 0;
+}
 
-    &:hover {
-        background-position-x: 100%;
-        color: var(--black);
-        text-decoration: none;
-    }
+article {
+    @include gradient-links;
```

Comment:
Gradient link hover moved to a separate mixin. This allows us to only apply this effect where expected (ex. apply to all `article`s on the site, and manually apply the mixin for other placements)